### ISSUE TITLE
adjust in-app keyboard size

### ISSUE THREAD

**Issue by Piping** [kprinssu/anne-keyboard-windows#23]
Hi,

First of all, thank you for this awesome project that enables me to customize the keyboard through windows. 

Can you show me how to change the size of keyboard shown in app? The keys are clustered together and the labels are not completely visible.

**Comment by ethanmsmith** [comment]
There is currently no way to resize the keyboard. Might be something worth looking into in the future, though. (Maybe when odins updates the keyboard firmware)

**Comment by kprinssu** [comment]
@Piping if you are referring to the UI, then it is doable. 
I am aware that the UI is clunky right now, but I do not have the time to redo it. If someone else is willing to do it, I can merge it and do a new build.

**Comment by Piping** [comment]
@kprinssu yeah, I am talking about UI. I never look at windows development and don't know C#. But if you can give me a start point on where the code should be updated, that will be very helpful.

**Comment by ethanmsmith** [comment]
@Piping I've found some free time, so I'm actually working on updating the ui at the moment. I've done some work, including enlarging the keyboard. You can see what I'm working on in my local repo

**Comment by Piping** [comment]
@ethanmsmith  Thanks!!

**Comment by kprinssu** [comment]
@ethanmsmith please submit a PR, I'll merge the changes in.

**Comment by ethanmsmith** [comment]
Its not complete, yet. I had to move some things around and haven't got them fully up and running again. If I am going to put in the effort to separate views, I don't want to half ass the connection aspect.

The UI is complete, but it is not back to full functionality, then I have to test. Do you have any tests you run builds against?

**Comment by kprinssu** [comment]
There aren't any automated tests and builds.

**Comment by ethanmsmith** [comment]
Ok. Good to know

**Comment by LMayne1** [comment]
Any idea why the light setting would only work on the first 7 keys off the KB? only escape through 7 are lighting up when i sync the profiles.

**Comment by kprinssu** [comment]
It's a bug that @ethanmsmith fixed. The fix is not added to the current release.

I am intending to do a new build later this week/early next week.

**Comment by ghost** [comment]
Github rookie here (which I why I post here): 
Is it possible to choose "no color"/transparent for the keycaps in this app?

PS: Really love the work here! 👍

**Comment by ethanmsmith** [comment]
What do you mean? Do you mean no lighting on a specific key/keys?
If so, I don't actually recall. My gut says if you set it to black it might... But I don't think there is anything that allows off backlighting

**Comment by kprinssu** [comment]
@ethanmsmith I have pushed a new release containing your changes.
@LMayne1 please give 1.0.4 a try.
@bravobavo setting the colour to black should turn off the backlight.

**Comment by ghost** [comment]
I was thinking the same thing when turning on black. And yes, turning 
off backlight on spesific keys was what I meant :=)
However, lately I think an issue has occured: when I set the buttons to 
black rgb(0,0,0), they always have this tint of the color that is 
currently selected (for example dark purple instead of black). First I 
thought this was because of the lighting spread from other buttons who 
were purple, but later, after changing everything to black it still was 
dark purple (or dark green if green was the selected color on the cirle 
picker). I do not recall having this problem before when testing, but 
maybe it is an issue, just wanted to bring it up ;)
